pkg/rancher-desktop/assets/scripts/configure-image-allow-list

```diff
@@ -18,9 +18,11 @@ mkcert -install
 mkcert localhost
 chown -R nobody:nobody $CAROOT
 
-# Configure containerd to pull images via OpenResty proxy.
-sed -i 's/-e HTTPS_PROXY.*--stderr/--stderr/' /etc/init.d/containerd
-sed -i 's/--stderr/-e HTTPS_PROXY=http:\/\/127.0.0.1:3128 --stderr/' /etc/init.d/containerd
+# configure proxy only when image-allow-list exists
+ial=/usr/local/openresty/nginx/conf/image-allow-list.conf
+setproxy="[ -f $ial ] && supervise_daemon_args=\"-e HTTPS_PROXY=http://127.0.0.1:3128 \$supervise_daemon_args\""
+sed -i "\#-f $ial#d" /etc/init.d/containerd
+sed -i "/^supervise_daemon_args/a $setproxy" /etc/init.d/containerd
 
 # Make sure openresty is started before containerd
 sed -i -E '/openresty/!s/^([ \t]*need.*)/\1 openresty/' /etc/init.d/containerd
```

Comment:
Note that we still force containerd to need openresty even if it's off.

Reply:
That should have been deleted because we manually start/stop openresty now.

Removed & force-pushed...